src/test/java/pl/targosz/homework/ReturningTextLengthTest.java

```diff
@@ -0,0 +1,17 @@
+package pl.targosz.homework;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ReturningTextLengthTest {
+
+    @ParameterizedTest
+    @MethodSource("textLength")
```

Comment:
https://www.baeldung.com/parameterized-tests-junit-5

4.6. Methods

tutaj w @MethodSource nie podajesz tej metody, którą chcesz przetestować, a metode, która zwraca argumenty wejściowe dla testu. To troszkę bez sensu by było, aby tutaj podawać metodę'textLength', bo przeciez wywołujesz ją tutaj 
![image](https://user-images.githubusercontent.com/17989854/137305981-e3bd59ad-d8dd-41a3-95cc-79bd9ff98d06.png)


I czym jest input, skąd on ma się wziąć?